app/dashboard/views.py

```diff
@@ -351,6 +361,118 @@ def new_interest(request, bounty_id):
         if interest.pending:
             start_work_new_applicant(interest, bounty)
 
+        if bounty.event:
+            from chat.tasks import add_to_channel, create_channel, create_user, chat_driver
+            from django.utils.text import slugify
+            try:
```

Comment:
In general, we should limit the use of `try` blocks to places where we expect exceptions to happen, rather than wrapping huge chunks of code in try blocks "just to be safe"

Reply:
so you'd rather multiple blocks? or less generalized Exceptions?

Reply:
both are good